packages/valory/connections/test_tools/docker_images.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
```

Comment:
You cannot place this here - won't exist outside this repo for any package. Test tools is either in aea or package specific.

Reply:
aah. OK!